src/indieweb_utils/indieauth/profile.py

```diff
@@ -43,7 +43,11 @@ def get_profile(me: str) -> Profile:
 
         me = "https://jamesg.blog"
 
-        profile = indieweb_utils.get_profile(me)
+        try:
+            profile = indieweb_utils.get_profile(me)
+        except indieweb_utils.ProfileError as exception:
```

Comment:
I think it would be clearer if, rather than showing it at a code example, if instead we used the raises notation in the docstring, above the example. This will document the exceptions _and_ allow IDEs to hook into it better.

i.e.

```
:raises:
    ProfileError: Profile could not be retrieved.
```

Reply:
I did not know this notation existed! This would be much more convenient. I shall make the requisite changes.